src/mondeath.cpp

```diff
@@ -165,6 +165,21 @@ void mdeath::acid(monster *z)
     g->m.add_field(z->pos(), fd_acid, 3, 0);
 }
 
+void mdeath::acidburst(monster *z) // makes monster w/ flag spawn acid upon death
+{
+    if (g->u.sees(*z)) {
```

Comment:
All changed/added lines should get padding on the inside of parentheses, like this:
`if( x )`